code/model/Comment.php

```diff
@@ -576,15 +576,15 @@ public function getCMSFields()
                 ->obj('Created')
                 ->scaffoldFormField($this->fieldLabel('Created'))
                 ->performReadonlyTransformation(),
-            TextField::create('Name', $this->fieldLabel('AuthorName')),
+            TextField::create('Name', $this->fieldLabel('Name')),
```

Comment:
There's no `'AuthorName'` field label in [`fieldLabels()`](https://github.com/janneklouman/silverstripe-comments/blob/master/code/model/Comment.php#L182), however there's one called `'Name'` with the default value of `'Author Name'`, so I reckoned it was just a typo to begin with, and therefor included this change so that the label can be translated. Maybe I'm missing something though.